label.rb

```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(title, color)
+    @id = Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @item << item unless @items.include?(item)
```

Comment:
Used `@item` instead of `@items`. This appears to be a typo as it should be `@items`. kindly fix that ``

Reply:
Great observation. Thank you for this. I'll be sure to correct it.